code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -36,7 +37,8 @@
 	spawn_blacklisted = TRUE
 	accompanying_object = /obj/item/clothing/head/space/syndicate
 	slowdown = LIGHT_SLOWDOWN
-	stiffness = HEAVY_STIFFNESS
+	stiffness = LIGHT_STIFFNESS
```

Comment:
Is this an intended change?

Reply:
From the changelog
balance: Captain's armor is somewhat harder to breach, and contractor space suits have become less stiff.